docs/dashboards-new/share-dashboards-outside-org.md

```diff
@@ -0,0 +1,117 @@
+---
+id: share-dashboard-outside-org
+---
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+# Share a Dashboard Outside Your Organization
+
+This page has information about sharing dashboards with users who do not have a Sumo Logic account. For information about sharing dashboards with other Sumo Logic users, see [Share a Dashboard](/docs/dashboards-new/share-dashboard-new).
+
+Dashboards often contain sensitive information, so typically you share them only with people within your organization. But sometimes you may want to allow people outside your organization to view a dashboard without logging in, for example, if your dashboard is displayed on a monitor in your operations center, or if you want to send an external person a link to a dashboard that reveals a critical problem.
+
+:::note
+Sharing outside an organization is always view-only and need auto-refresh enabled. Also, if this is the first time that auto-refresh is being enabled, it may take some time to backfill the data depending on your queries and timeranges. See [About Dashboards](about.md).
+:::
+
+You can share a dashboard with all external users (“the world”), or with a selected set of external users that you specify in the service allowlist. 
+
+Before you can share a dashboard with users external to your organization, you must enable a security policy, ensure that you have the right role capabilities, and depending on your option selection, enable the service allowlist.
+
+## Enable the Sumo Logic policy
+
+In this step you enable sharing of dashboards outside your organization. You must have Admin privileges to enable the policy.
+
+1. Go to **Administration** > **Security** > **Policies**.
+1. Select the **Enable** check box for **Share Dashboards Outside of the Organization**.
+1. The policy is changed. It can take up to 5 minutes for the policy to go into effect.
+
+## Verify your role allows external sharing
+
+You must have specific role capabilities to share a dashboard with external users:
+
+* **Share Dashboards with the Allowlist.** Share dashboards with users only on the allowlist.
+* **Share Dashboards with the World.** Share dashboards with external users. 
+
+If you do not already have a role that grants you the required capability, an Admin user can assign you one, as described on [Assign Roles to Users](/docs/manage/users-roles/roles/add-remove-users-role.md), or create a new one as necessary, as described on [Create a New Role](/docs/manage/users-roles/roles/create-manage-roles.md). 
+
+## Add external users to service allowlist
+
+If you want to share dashboards with specific external users, rather than the world, the service allowlist must be enabled, and you must provide the list of IPs or CIDRs that will be able to view dashboards without logging in. For instructions, see [Enable Dashboard Allowlist](/docs/manage/security/create-allowlist-ip-cidr-addresses.md).
```

Comment:
```suggestion
If you want to share dashboards with specific external users, rather than the world, the service allowlist must be enabled and you must provide the list of IPs or CIDRs that will be able to view dashboards without logging in. For instructions, see [Enable Dashboard Allowlist](/docs/manage/security/create-allowlist-ip-cidr-addresses.md).
```